### PR TITLE
[Lake] Support alter lake tablet in BE

### PR DESCRIPTION
**Pull request by wyb**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Like local tablets, there are three schema change types,
including linked, direct and sorted schema change.

In sorted schema change, use DeltaWriter to sort data.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wyb** [comment]
run all

**Comment by wyb** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wyb** [comment]
run starrocks_admit_test

**Review by sduzh — APPROVED** [review]

File: be/src/storage/lake/schema_change.cpp [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+}
+
+Status ConvertedSchemaChange::init() {
+    _read_params.reader_type = ReaderType::READER_ALTER_TABLE;
+    _read_params.skip_aggregation = false;
+    _read_params.chunk_size = config::vector_chunk_size;
+    _read_params.use_page_cache = false;
+
+    ASSIGN_OR_RETURN(auto base_tablet_schema, _base_tablet->get_schema());
+    _base_schema = ChunkHelper::convert_schema_to_format_v2(*base_tablet_schema,
```
Comment (chaoyli) [inline be/src/storage/lake/schema_change.cpp:113]: In the lake, format v1 is also not supported. It's not necessary to convert the format.

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: be/src/storage/lake/schema_change.cpp (hunk 1 above)
Comment (wyb) [reply be/src/storage/lake/schema_change.cpp:113]: Remove this later.

**Pull request merged**